python/ray/experimental/channel/cpu_nccl_group.py

```diff
@@ -0,0 +1,157 @@
+import asyncio
+from collections import defaultdict
+from typing import Optional, Tuple, List
+from unittest import mock
+
+import torch
+
+import ray
+import ray.experimental.channel as ray_channel
+from ray.experimental.channel.gpu_communicator import TorchTensorAllocator, ReduceOp
+
+
+@ray.remote(num_cpus=0)
+class Barrier:
```

Comment:
It would be better to give this a more descriptive name, now that it can be used in user apps. The name should include CPU, communicator, etc.

Reply:
Changed this to CPUCommunicator